homeassistant/components/mobile_app/__init__.py

```diff
@@ -90,3 +97,16 @@
         hass.config_entries.async_forward_entry_setup(entry, DATA_SENSOR))
 
     return True
+
+
+async def async_remove_entry(hass: HomeAssistantType,
+                             entry: ConfigEntry) -> None:
```

Comment:
Either this return type is incorrect or the `return` statement below is unnecessary